docs/internal/versioning.md

```diff
@@ -0,0 +1,11 @@
+# Versioning
+
+
+## Overview
+Daffodil uses Lerna and Nx to organize all of its projects in a single root project, or monorepo. There is one ``package.json`` in the root directory that contains the dependencies and versions used throughout the entire monorepo, and also separate ``package.json`` files throughout the monorepo for the dependencies of each sub-project.
```

Comment:
```suggestion
Daffodil uses Lerna and Nx to organize all of its projects in a single root project, or monorepo. There is one `package.json` in the root directory that contains the dependencies and versions used throughout the entire monorepo, and also separate `package.json` files throughout the monorepo for the dependencies of each sub-project.
```

For consistency, update the code blocks to single back ticks as that's what's used throughout the codebase.